requirements/requirements.txt

```diff
@@ -5,3 +5,5 @@ targ>=0.3.7
 inflection>=0.5.1
 typing-extensions>=4.3.0
 pydantic[email]==2.*
+tzdata>=2024.1
```

Comment:
Based on my experience with `zoneinfo` on `Windows` machines, it can exhibit some unexpected behavior even after installing [tzdata](https://docs.python.org/3/library/zoneinfo.html#data-sources). If `piccolo` aims to support cross-platform conditions, I suggest running continuous integration tests for timestamp-related functionality in the `Windows` container, at least for a period, once this PR is merged.

Reply:
@dantownsend @sinisaos This might be a little off-topic, but have you guys heard about [uv](https://github.com/astral-sh/uv)? I've used it for a couple of personal projects and it looks promising to me.

Reply:
@jrycw Sorry, I've never used `uv`. I always use `virtualenv` and I'm used to it.